### DISCUSSION TITLE
PCB

### DISCUSSION THREAD

**Discussion by wolffman122** [HASwitchPlate/openHASP#356] [Hardware]
I was wondering if anyone has made a pcb similar to this [one](https://github.com/HASwitchPlate/HASPone/tree/main/PCB).  Don't really care if the screen still needs wires to attach to the PCB.  Looking to put into a gang for switches in my house.

**Comment by bensuffolk** [comment]
I’d suggest getting a Lanbon L8. It’s all done for you and comes in a professional looking plate.
